docs/readmes/bazel/bazel.md

```diff
@@ -0,0 +1,510 @@
+---
+id: bazel_basics
+title: AGW with bazel
```

Comment:
"Bazel" vs "bazel" - is this consistent (also in existing docs)? - I would have used "Bazel" because I interpret it as a proper noun - what do you think?

Reply:
- [ ] Official Bazel docs use `Bazel` - so let's use that.

- [ ] Also since you comment here - the `id` and `title` need to be made the same.